src/App.tsx

```diff
@@ -143,7 +143,8 @@ function AppInternal() {
 }
 
 function App() {
-  const network = (window.location.host.indexOf("ropsten") > -1) ? Network.ROPSTEN : Network.MAINNET;
+  // url for atk on ropsten: https://allthekeeps.test.keep.network
+  const network = (window.location.host.indexOf("test") > -1) ? Network.ROPSTEN : Network.MAINNET;
```

Comment:
Could the network type be set somehow on build?

Reply:
9399eb6